src/qml/MainView.qml

```diff
@@ -10,13 +10,12 @@ Control {
     id: mainView
 
     property var attachmentPreview: undefined
-    property int keyboardHeight: 0
     readonly property var manager: app.stateManager
 
     RowLayout {
         anchors {
             fill: parent
-            bottomMargin: (logControl.visible ? logControl.height : 0) + keyboardHeight
+            bottomMargin: keyboardHandler.keyboardHeight
```

Comment:
logControl logic is broken, restore deleted addition

Reply:
Ok